plugins/services/src/js/components/DeclinedOffersTable.js

```diff
@@ -210,12 +210,16 @@ class DeclinedOffersTable extends React.Component {
 
           return (
             <Tooltip content={hostname} maxWidth={400} wrapText={true}>
-              <Link
-                className="table-cell-link-primary"
-                to={`/nodes/${node.id}`}
-              >
-                {hostname}
-              </Link>
+              {node && node.id ? (
+                <Link
+                  className="table-cell-link-primary"
+                  to={`/nodes/${node.id}`}
+                >
+                  {hostname}
+                </Link>
+              ) : (
+                hostname
```

Comment:
Can we wrap this in `<span className="table-cell-link-primary" />`? That way it looks the same as when it has an id and gets linked.

If it was intentional to make it look different, I think we need to make the difference more noticeable than just changing the font weight - maybe lighten the text.